pkg/apiserver/rest/rest_server.go

```diff
@@ -142,6 +147,9 @@ func (s *restServer) setupLeaderElection() (*leaderelection.LeaderElectionConfig
 			OnStartedLeading: func(ctx context.Context) {
 				go velasync.Start(ctx, s.dataStore, restCfg, s.usecases)
 				s.runWorkflowRecordSync(ctx, s.cfg.LeaderConfig.Duration)
+				if !s.cfg.DisableStatisticCronJob {
+					collect.StartCalculatingInfoCronJob(s.dataStore)
```

Comment:
It should be noted that the function is not controlled by context.

Reply:
Have considered this issue. Currently when api-server shut-down the cornJob routine will be closed.